README.md

```diff
@@ -60,10 +60,10 @@ So we are dependent on the browser supporting the manifests.
 
 ### Shaka Player Embedded (for native iOS)
 
-We have another project called [Shaka Player Embedded][] that offers the same
-features and similar APIs for native apps on iOS.  This project uses its own
+We have another project called [Shaka Player Embedded][] which offers the same
```

Comment:
"That" and "which" sounded the same to my ear, so I looked it up.  It's a subtle difference, but you might be right that this is somewhat more correct grammatically.

https://www.grammarly.com/blog/which-vs-that/

That said, I don't think that picking over minor nuances of our grammar is the most valuable way you could contribute.  Have you considered other ways to contribute to the documentation?  What about writing a new tutorial?  Or perhaps you could review the docs to help identify areas that are unclear or missing high-level docs entirely?

Reply:
Ok Sure

Reply:
BTW I want to make some good technical contributions so bit of guidelines would be really helpful.

Reply:
I don't know what your background is, but we have a few tags that you might want to look at:

 - https://github.com/google/shaka-player/labels/contributions%20welcome
   - These are issues the core team is not working on at all right now
 - https://github.com/google/shaka-player/labels/easy%3F
   - These are issues someone thought might be relatively easy to start on
 - https://github.com/google/shaka-player/labels/docs
   - These are docs issues

Feel free to comment on an issue you're interested in if you'd like to propose a solution or have it assigned to you.  We're happy to discuss with you.